python/lsst/l1dbproto/l1dbschema.py

```diff
@@ -262,11 +309,25 @@ def getAfwSchema(self, table_name, columns=None):
             else:
                 units = column.unit or ""
                 # some units in schema are not recognized by afw but we do not care
-                key = schema.addField(afw_col,
-                                      type=self._afw_type_map_reverse[column.type],
-                                      doc=column.description or "",
-                                      units=units,
-                                      parse_strict="silent")
+                if self._afw_type_map_reverse[column.type] == 'String':
+                    key = schema.addField(afw_col,
+                                          type=self._afw_type_map_reverse[column.type],
+                                          doc=column.description or "",
+                                          units=units,
+                                          parse_strict="silent",
+                                          size=10)
+                elif afw_col == 'coord_ra' or afw_col == 'coord_dec':
+                    key = schema.addField(afw_col,
+                                          type='Angle',
+                                          doc=column.description or "",
+                                          units=units,
```

Comment:
Should it be "rad" for units, whould it confuse afw.table if you leave "deg" here?

Reply:
It doesn't look like SourceTable cares what you set for units in the key. The type seems to always display as radians until you ask for Angle.asDegrees(). When you request the column an lsst.geom.Angle object is always returned. I can remove the unit from this key definition for clarity if that satisfies.

Reply:
I think it's better to remove it then.